notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>This would be hard to do with astroquery. There's no way to search for m-dwarfs, let alone high proper-motion ones.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>